controllers/spiaccesstokenbinding_controller.go

```diff
@@ -128,6 +131,18 @@ func (r *SPIAccessTokenBindingReconciler) Reconcile(ctx context.Context, req ctr
 		return ctrl.Result{}, nil
 	}
 
+	// cleanup bindings by lifetime
+	bindingLifetime := time.Since(binding.CreationTimestamp.Time).Seconds()
+	if bindingLifetime > r.Configuration.AccessTokenBindingTtl.Seconds() {
+		err := r.Client.Delete(ctx, &binding)
+		if err != nil {
+			lg.Error(err, "failed to cleanup binding on reaching the max lifetime", "binding", binding.ObjectMeta.Name, "error", err)
```

Comment:
no need to specify the `binding` in the logger, it is already done by the controller runtime.

Reply:
ok for all